android/src/main/java/io/radar/capacitor/RadarPlugin.java

```diff
@@ -785,6 +895,95 @@ public void onComplete(@NotNull Radar.RadarStatus status, @Nullable RadarRoutes
         }
     }
 
+    @PluginMethod()
+    public void sendEvent(final PluginCall call) throws JSONException  {
+        String customType = call.getString("customType");
```

Comment:
Let's `reject` if `customType` is missing

Reply:
@tjulien `customType` can't be missing, Typescript checks